docs/dev/reference/blocks_api.md

```diff
@@ -138,9 +138,10 @@ Block data has two subsets: logged and stored data.
 
 Logged data consists of the fields that are published to the history service.
 This does not include string values, links, arrays, and some of the more persistent settings.
+You can't recreate a block's settings by writing its logged output.
```

Comment:
recreate settings?
using mode logged with a write is an error and the API should throw an error